packages/neuvector/generated-changes/overlay/questions.yaml

```diff
@@ -1,11 +1,5 @@
 questions:
 #image configurations
-- variable: registry
-  default: "docker.io"
-  description: image registry
-  type: string
-  label: Image Registry
-  group: "Container Images"
```

Comment:
Sort of surprised this got backported into 2.6. I know we made this switch in 2.7, just want to make sure we are doing this purposefully in 2.6.

Reply:
@cbron In 2.6 also, NeuVector is not using registry from questions.yaml. It is a bug, so removed from questions.yaml.  Registry need to be set only by systemDefaultRegistry. 
https://github.com/selvamt94/charts/blob/dev-v2.6/charts/neuvector/100.0.2%2Bup2.2.3/templates/controller-deployment.yaml#L59

Reply:
Ok makes sense